src/components/radio/radio.module.scss

```diff
@@ -94,69 +105,9 @@
   }
 }
 
-.light {
-  &.toggler {
-    width: 16px;
-    height: 16px;
-    min-width: 16px;
-    border-color: var(--rp-ui-kit-e-200);
-    background-color: var(--rp-ui-kit-bg-000);
-
-    &::after {
-      width: 8px;
-      height: 8px;
-    }
-  }
-
-  &.children-container {
-    max-width: 100%;
-  }
-
-  &:hover:not(.disabled) {
-    border-color: var(--rp-ui-kit-e-400);
-  }
-
-  &:hover:after:not(.disabled) {
-    background-color: var(--rp-ui-kit-topaz-hover);
-  }
-
-  &.disabled {
-    .toggler {
-      opacity: 0.3;
-    }
-
-    &::after {
-      opacity: 0.3;
-    }
-  }
-}
-
 .dark {
   &.toggler {
```

Comment:
You are removed the `light` mode, can the `dark` mode now be removed as well? To manage color scheme using themes.